src/components/controls/InfoPanelControl.tsx

```diff
@@ -1,23 +1,25 @@
 import React from 'react'
 import { useMap } from 'react-leaflet'
-
 import { Stakeholder } from 'types'
 import ExternalLinkSvg from 'assets/external_link.svg'
 
 interface InfoPanelControlProps {
   stakeholder: Stakeholder | null
   onClose: () => void
+  onClick: (e: React.MouseEvent<HTMLDivElement>) => void
```

Comment:
The onClick function isn't being called in InfoPanelControl. I'm not familiar with 
 the functionality of the TierGater click. Could you give me an overview of it?